truss/tests/templates/server/test_model_wrapper.py

```diff
@@ -177,6 +177,7 @@ async def mock_predict(return_value, request: Request):
         ):
             model_wrapper = model_wrapper_class(config, sdk_trace.NoOpTracer())
             model_wrapper.load()
+            time.sleep(15)  # Wait for underling model to load
```

Comment:
This worked locally for me, but consistently ran into 'model not loaded' errors in CI. This doesn't go through the docker flow that other tests do, so I think it's just a little bit of flakiness depending on how fast it happens to load.

I don't love introducing sleeps, but if it makes this test more consistent, it seems worthwhile in the short term